src/components/organisms/Tabs/TabButtons/TabButtons.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, FC, ChangeEvent, useCallback } from 'react';
+import { useTabsContext } from '../hooks/useTabsContext';
+import { useViewport } from '../../../../hooks/useViewport';
+import Dropdown from '../../../atoms/Dropdown/Dropdown';
+import { grid, colors } from '../../../../constants';
+import TabButton, { TabButtonProps } from '../TabButton/TabButton';
+import { DropdownOption } from '../../../..';
+import styled from 'styled-components';
+
+interface TabButtonsProps {
+  tabButtons: TabButtonProps[];
+  defaultTab?: string;
+  dataAutomation?: string;
```

Comment:
not necessarily for this PR - not sure what other people think but I'm actually thinking its better to just have the actual prop as `["data-automation"]` for consistency with the underlying element